src/System.Security.Cryptography.Csp/tests/CspParametersTests.cs

```diff
@@ -33,5 +34,89 @@ public static void SetFlags_ValidatesInput()
             Assert.Throws<ArgumentException>(
                 () => cspParameters.Flags = (CspProviderFlags)0x0100 | CspProviderFlags.NoPrompt);
         }
+
+        [Fact]
+        public static void KeyPassword_SetGet()
+        {
+            var cspParameters = new CspParameters();
+            using (var pwd = new SecureString())
+            {
+                pwd.AppendChar('p');
+                cspParameters.KeyPassword = pwd;
+                Assert.Same(pwd, cspParameters.KeyPassword);
+            }
+        }
+
+        [Theory(Skip = "Manual test - requires Smart Card - read instructions")]
```

Comment:
This is still marked [Theory].  Please comment out the attribute instead of marking it Skip